### ISSUE TITLE
STS -> MN: Correct encoding of main titles

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/mnconvert#83]
*In relation to https://github.com/metanorma/metanorma-bsi/issues/30*

I've realized that document titles are being incorrectly encoded in several cases.
I understand that some documents don't require to have a *part* title.

For example....

From *isoiec-29151*:
```xml
<intro>Information technology</intro>
<main>Security techniques — Code of practice for personally identifiable information protection</main>
<full>Information technology — Security techniques — Code of practice for personally identifiable information protection</full>
```

Generated adoc:
```adoc
:title-intro-en: Information technology
:title-main-en: Security techniques
:title-part-en: Code of practice for personally identifiable information protection
```

Should have been encoded without using `:title-part-en:` because XML source doesn't contain it:, i.e.:
```adoc
:title-intro-en: Information technology
:title-main-en: Security techniques — Code of practice for personally identifiable information protection
```

Now, the cases where we have to add a part title are those where `<compl>` tag is indicated.

For example...

From *iso-10667-1*:
```xml
    <intro>Assessment service delivery</intro>
    <main>Procedures and methods to assess people in work and organizational settings</main>
    <compl>Requirements for the client</compl>
    <full>Assessment service delivery — Procedures and methods to assess people in work and organizational settings — Part 1: Requirements for the client</full>
```

Generated adoc:
```adoc
:title-intro-en: Assessment service delivery
:title-main-en: Procedures and methods to assess people in work and organizational settings
:title-part-en: Part 1: Requirements for the client
```
(here, we need to strip "Part1: " text) 

Also, there are cases where an *intro* title is not given in XML source, so we don't need to use `:title-intro-en:`.

For example...

From *bs-8654-2015*:
```xml
<main>Domestic and hospitality use ceramic tableware articles intended for contact with foodstuffs</main>
<compl>Specification</compl>
<full>Domestic and hospitality use ceramic tableware articles intended for contact with foodstuffs – Specification</full>
```

Generated adoc:
```adoc
:title-intro-en: Domestic and hospitality use ceramic tableware articles intended for contact with foodstuffs
:title-part-en: Specification
```

Should have been encoded as:
```adoc
:title-main-en: Domestic and hospitality use ceramic tableware articles intended for contact with foodstuffs
:title-part-en: Specification
```

In summary:
`<intro>` xml tag indicates `:title-intro-*:`
`<main>` xml tag indicates `:title-main-*:`
`<compl>` xml tag indicates `:title-part-*:`
